website/src/pages/Community/index.tsx

```diff
@@ -1,5 +1,5 @@
 import {
-  Button, Grid, Link, Paper, Typography,
+  Button, Grid, Paper, Typography,
```

Comment:
Why is this change needed?

Reply:
This change is needed, because the deploy and the build got failed. In the deploy log, it was suggested to remove this link as this link is related to community events.